server/schedulers/hot_region.go

```diff
@@ -515,11 +533,9 @@ func (bs *balanceSolver) solve() []*operator.Operator {
 	}
 
 	bs.cur = &solution{}
-	tryUpdateBestSolution := func(isUniformFirstPriority bool) {
-		if bs.cur.progressiveRank == -1 && isUniformFirstPriority {
-			// Because region is available for src and dst, so stddev is the same for both, only need to calcurate one.
-			// If first priority dim is enough uniform, -1 is unnecessary and maybe lead to worse balance for second priority dim
-			hotSchedulerResultCounter.WithLabelValues("skip-uniform-store", strconv.FormatUint(bs.cur.dstStore.GetID(), 10)).Inc()
+	tryUpdateBestSolution := func() {
+		if label, ok := bs.filterUniformStore(); ok {
+			schedulerCounter.WithLabelValues(bs.sche.GetName(), fmt.Sprintf("%s-skip-%s-uniform-store", bs.rwTy.String(), label)).Inc()
```

Comment:
Why not still using hotSchedulerResultCounter

Reply:
The previous monitoring location is wrong